search/lib/cmr.js

```diff
@@ -251,28 +269,24 @@ async function convertParams (providerId, params = {}) {
     });
     logger.debug(`Params: ${JSON.stringify(params)}`);
```

Comment:
done with this?

Reply:
I think this is also going to be helpful to keep as debug logs for UAT/SIT